README.md

```diff
@@ -735,16 +735,18 @@ maintaining the Node.js project.
   **Frank Qiu** <<iam.frankqiu@gmail.com>> (he/him)
 * [marsonya](https://github.com/marsonya) -
   **Akhil Marsonya** <<akhil.marsonya27@gmail.com>> (he/him)
+* [Mesteery](https://github.com/Mesteery) -
```

Comment:
Wasn't the original position correct?

Reply:
Oof, it might've been, I tried to organize it while in markdown mode, and I may have made a mistake. I'll proof check it again shortly. 

Sorry again